tests/core/nogo/custom/return_bool_analyzer.go

```diff
@@ -0,0 +1,37 @@
+// returnboolanalyzer checks for functions that return bool.
+package returnboolanalyzer
+
+import (
+	"go/ast"
+
+	"golang.org/x/tools/go/analysis"
+)
+
+const doc = `report functions that return booleans
+
+The return_bool_analyzer analyzer reports functions that return booleans.`
+
+var Analyzer = &analysis.Analyzer{
+	Name: "return_bool_analyzer",
```

Comment:
Don't use underscores here. Also, analyzer is redundant.

`Name: "boolreturn"`